sql-db/hibernate-reactive/src/main/java/io/quarkus/ts/reactive/PanacheEndpoint.java

```diff
@@ -0,0 +1,135 @@
+package io.quarkus.ts.reactive;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+
+@Path("/library")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class PanacheEndpoint {
+
+    @Inject
+    AuthorRepository authors;
+
+    @GET
+    @Path("books")
```

Comment:
```suggestion
    @Path("/books")
```
applies to all endpoints in this class

Reply:
But why? This path is a relative one, so a slash in the beginning would be misleading.

Reply:
It is not misleading at all, look how other endpoints define it in other modules

Reply:
Well, I can fix this in other modules too, if such uniformity is required :D

Reply:
:D Ok, both styles can be used. Just a matter of preference, resolving this and similar threads